ui/app/adapters/sync/association.js

```diff
@@ -7,15 +7,13 @@ import ApplicationAdapter from 'vault/adapters/application';
 import { assert } from '@ember/debug';
 
 export default class SyncAssociationAdapter extends ApplicationAdapter {
-  namespace = 'v1/sys/sync/destinations';
+  namespace = 'v1/sys/sync';
```

Comment:
removed `/destinations` because in addition to the request for destination sync statuses if we want to extend these requests (i.e list all association counts in the overview page) that request also does not have `/destinations` in the path [docs](https://github.com/hashicorp/vault/pull/23975/files#diff-ccc5cfb41462aaf17193c92f2919697b6373c0b8fcee8227c3d3658adfeef9a2R601)